internal/backend/db/dbgorm/deployments.go

```diff
@@ -108,11 +108,11 @@ func (db *gormdb) listDeploymentsWithStats(ctx context.Context, filter sdkservic
 	COUNT(case when sessions.current_state_type = ? then 1 end) AS error,
 	COUNT(case when sessions.current_state_type = ? then 1 end) AS completed,
 	COUNT(case when sessions.current_state_type = ? then 1 end) AS stopped
-	`, sdktypes.SessionStateTypeCreated.ToProto(),
-		sdktypes.SessionStateTypeRunning.ToProto(),
-		sdktypes.SessionStateTypeError.ToProto(),
-		sdktypes.SessionStateTypeCompleted.ToProto(),
-		sdktypes.SessionStateTypeStopped.ToProto()).
+	`, int32(sdktypes.SessionStateTypeCreated.ToProto()),
```

Comment:
comment why

Reply:
because toProto won't return int32, but ~int32, e.g. int32 compatible type. So in gorm and PG drive will refuse to save it in bigint column

Reply:
I think @itayd meant that you should comment that in the code.

Reply:
done